.github/workflows/conda-package.yml

```diff
@@ -800,7 +802,8 @@ jobs:
         with:
           run-post: false
           channel-priority: "disabled"
-          channels: conda-forge,nodefaults
+          channels: conda-forge
```

Comment:
`upload_windows` job uses `conda-incubator/setup-miniconda` step, but there is `channels` property missing. I think it needs to be aligned with other usages of `conda-incubator/setup-miniconda` step and so both `channels: conda-forge` and `conda-remove-defaults: true` properties to be defined there.

Reply:
Otherwise the base environment is creating with packages from `pkgs/main` channel which is restricted.